library/src/main/java/com/shawnlin/numberpicker/NumberPicker.java

```diff
@@ -1804,7 +1810,27 @@ protected void onDraw(Canvas canvas) {
                 if (!isHorizontalMode()) {
                     textY += getPaintCenterY(mSelectorWheelPaint.getFontMetrics());
                 }
-                drawText(scrollSelectorValue, x, textY, mSelectorWheelPaint, canvas);
+
+                int xOffset = 0;
+                int yOffset = 0;
+
+                if (i != mWheelMiddleItemIndex) {
```

Comment:
It would be better to replace with `if (i != mWheelMiddleItemIndex && mItemSpacing != 0)`